runtime/src/transaction_batch.rs

```diff
@@ -1,21 +1,37 @@
 use {
     crate::bank::Bank, solana_sdk::transaction::Result,
-    solana_svm_transaction::svm_transaction::SVMTransaction, std::borrow::Cow,
+    solana_svm_transaction::svm_message::SVMMessage,
 };
 
+pub enum OwnedOrBorrowed<'a, T> {
+    Owned(Vec<T>),
+    Borrowed(&'a [T]),
+}
+
+impl<T> core::ops::Deref for OwnedOrBorrowed<'_, T> {
```

Comment:
nit: `use std::ops::Deref` at the top of this file?

Reply:
[b7e4f86](https://github.com/anza-xyz/agave/pull/2837/commits/b7e4f860286b75b7f26cec867078c2a599a9b253)